build.psm1

```diff
@@ -140,9 +140,10 @@ function Get-EnvironmentInformation
         $environment += @{'IsFedora' = $LinuxInfo.ID -match 'fedora' -and $LinuxInfo.VERSION_ID -ge 24}
         $environment += @{'IsOpenSUSE' = $LinuxInfo.ID -match 'opensuse'}
         $environment += @{'IsSLES' = $LinuxInfo.ID -match 'sles'}
+        $environment += @{'IsRedHat7' = $LinuxInfo.ID -match 'rhel' -and $LinuxInfo.VERSION_ID -match '7' }
```

Comment:
Perhaps the logic should be more like Fedora with a `-ge` rather than a `-match` so we don't have to fix it again

Reply:
I modelled it after CentOS and others, where we are matching for the VERSION_ID. And since the variable is called `IsRedHat7` we may want to revisit when RedHat goes to version `8` anyways.

Reply:
Perhaps we should have a test at the end of the detection that verifies we matched at least one and throw if we don't.